src/types/Event/EventStatsWrapper/interface.ts

```
@@ -0,0 +1,9 @@
+/**
+ * Interface for EventStatsWrapper component props.
+ */
+export interface InterfaceEventStatsWrapperProps {
+    /**
```

ESLint

[error] Delete `··`

(prettier/prettier)